contrib/gopls/update.py

```diff
@@ -1 +1,3 @@
-pattern = r"/tags/gopls/v([\d.]+)\.tar\.gz"
+url = "https://api.github.com/repos/golang/tools/git/refs/tags"
+pattern = r"refs/tags/gopls/v([\d.\w-]+)"
```

Comment:
I could add a double quote to the end of the regex to get rid of matching the `\w-` and the `ignore` below, but idk

Reply:
Also the `refs` isn't strictly necessary, can remove it if you want